salvo/src/lib/benchmark/base_benchmark.py

```diff
@@ -0,0 +1,68 @@
+"""
+Base Benchmark object module that contains
+options common to all execution methods
+"""
+import os
+import logging
+
+import src.lib.docker_image as docker_image
+import src.lib.docker_volume as docker_volume
+
+log = logging.getLogger(__name__)
+"""
+Base Benchmark class with common functions for all invocations
+"""
+
+
+def get_docker_volumes(output_dir, test_dir=None):
```

Comment:
Assuming the test_dir is meant to be a string, can we try to also keep its default value within the same object type? Even though Python isn't type safe, it is good to follow the generic type safety rules to avoid unexpected programming errors, like when someone tries to call a string method on a NoneType.

```py
def get_docker_volumes(output_dir, test_dir=''):
```

Reply:
sure